app/models/todo_list.rb

```diff
@@ -1,4 +1,14 @@
 class TodoList < ActiveRecord::Base
+  has_and_belongs_to_many :users
   has_many :todo_items, -> { order("position ASC") }, dependent: :destroy
   accepts_nested_attributes_for :todo_items, allow_destroy: true
+  accepts_nested_attributes_for :users 
+  def completed?(user)
+  	self.todo_items.each do |todo_item|
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Redundant `self` detected.<br>Tab detected.